packages/server/src/koaJsonRpc/lib/utils.ts

```diff
@@ -19,8 +19,33 @@
  */
 
 import type { Server } from 'http';
+import constants from '@hashgraph/json-rpc-relay/dist/lib/constants';
+
+export function hasOwnProperty(obj: any, prop: PropertyKey): boolean {
+  return Object.prototype.hasOwnProperty.call(obj, prop);
+}
 
 export function setServerTimeout(server: Server): void {
   const requestTimeoutMs = parseInt(process.env.SERVER_REQUEST_TIMEOUT_MS ?? '60000');
   server.setTimeout(requestTimeoutMs);
 }
+
+export function getBatchRequestsMaxSize(): number {
```

Comment:
Q: With these methods why required a cast every time it's called. These seem to be pure in nature so why not do once on class instantiation and then returned the stored const value?

Reply:
UPDATE: Adding this change breaks some of the integration and acceptance tests because they rely on modifying process env variables dynamically to verify the behaviour.

Reply:
🤦🏾‍♂️ that's unfortunate.
Can you open a ticket for us to come back to this and resolve it.
Seems very inefficient and test code shouldn't bloc this.
We might need to reassess that approach then.

Reply:
We only call these methods when constructing the `KoaJsonRpc` object and it is instantiated only once in the beginning of `server.ts`, so this shouldn't have any impact on performance currently. 

I agree it will be a nice improvement, though, to have these env variables captured in const values and then reusing those, I will open an issue for this.

Reply:
Done, https://github.com/hashgraph/hedera-json-rpc-relay/issues/2686.